components/sensor/pulse_width.rst

```diff
@@ -7,7 +7,12 @@ Pulse Width Sensor
 
 The ``pulse_width`` sensor allows you to measure how long a given digital signal
```

Comment:
Could you add mode detailed information here, please? My usecase is exactly what is described here, but it doesn't work.

So, I want to measure water consumption by multiplying of the pump runtime by the flow. Pump runtime is the length of the signal HIGH. But this reported very unrelated data. Mostly not reacting on the pulse, otherwise reporting super high value.

Reply:
Please keep your specific use-case to the issue that you've opened for it.
This is a PR to add generic clarification and missing information to the docs, not to discuss why it does not work for your usecase.